test/extended/testdata/build-extended/bc-scripts-by-url.json

```diff
@@ -0,0 +1,39 @@
+{
+  "kind": "BuildConfig",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "java-extended-build-from-url"
+  },
+  "spec": {
+    "source": {
+      "git": {
+        "uri": "https://github.com/php-coder/java-maven-hello-world.git"
```

Comment:
I didn't find where we're using https://github.com/bparees/javaee7-hol in origin and project looks a bit abandoned. Also it looks like overkill for a such simple test. We're testing how is OpenShift building the images and not images itself. I believe that console "hello world" application should be enough. Another idea is that keeping things simple could allow to us use this example as a demonstration of the extended builds functionality.

Are you sure that I should use another example?